pkg/network/cache/infocache.go

```diff
@@ -114,6 +126,27 @@ func newPodInterfaceCacheStore(vmi *v1.VirtualMachineInstance, baseDir, pattern
 	return podInterfaceCacheStore{vmi: vmi, baseDir: baseDir, pattern: pattern}
 }
 
+type vmNetworkingInfraInterfacesStore struct {
+	pid     string
+	pattern string
+	baseDir string
+}
+
+func (d vmNetworkingInfraInterfacesStore) Read(iface string) (*networkdriver.VIF, error) {
+	cachedIface := &networkdriver.VIF{}
+	err := readFromCachedFile(cachedIface, getInterfaceCacheFile(d.baseDir, d.pattern, d.pid, iface))
+	return cachedIface, err
+}
+
+func (d vmNetworkingInfraInterfacesStore) Write(ifaceName string, ifaceToCache *networkdriver.VIF) (err error) {
+	err = writeToCachedFile(ifaceToCache, getInterfaceCacheFile(d.baseDir, d.pattern, d.pid, ifaceName))
+	return
```

Comment:
missing returning error.

Reply:
Nope, look at the return in the method signature. I don't like at all named return values, but, the others are like this.

Reply:
puff, I don't like it a bit can we derail a little from the others ?

Reply:
Stopped using a named return value.